sigstore-conformance/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id("java")
+    id("application")
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation(project(":sigstore-java"))
```

Comment:
curious if we should we also be testing against the latest release of sigstore-java? Since this is just at head.

Reply:
Hmm -- I don't think there's any problem with testing against the latest release, but I'm not sure it would add a ton of value either: the conformance suite mostly ensures that individual changes don't break the overall behavioral contract expected of Sigstore clients, so we'd expect a release that has already passed it once to continue passing it.

(We'd also end up de facto testing the latest release each time, by virtue of testing its `HEAD` commit right before the release is made. But we could make that explit/double it by also adding an event to test on the tag as well!)

Reply:
I also think it makes more sense to test at head so you can detect non-conformant behaviour as soon as possible.